src/interface_service/requirements.txt

```diff
@@ -9,4 +9,6 @@ dynaconf==3.1.9
 jinja2==3.1.2
 minio==7.1.8
 google-cloud-pubsub==2.13.0
+joblib==0.14.1
+scikit-learn==0.24.1
```

Comment:
Version 0.24.1 is incompatible with Python 3.10 and makes the Docker image building fail.